app.js

```diff
@@ -23,8 +29,14 @@ app.use((err, req, res, next) => {
   }
 });
 
+// app.use((err, req, res, next) => {
```

Comment:
Make sure you remove any commented out code - especially if the code may be on a future ticket :)